modules/core/client/components/LanguageSwitch.component.js

```diff
@@ -1,33 +1,153 @@
-import React from 'react';
+// External dependencies
+import { Modal } from 'react-bootstrap';
+import { useTranslation } from 'react-i18next';
+import classnames from 'classnames';
 import PropTypes from 'prop-types';
-import LanguageSwitchContainer from './LanguageSwitchContainer';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+// Internal dependencies
 import {
-  LanguageSwitchDropdown,
-  LanguageSwitchSelect,
-} from './LanguageSwitchPresentational';
+  getLocales,
+  getSearchedLocales,
+} from '@/modules/core/client/utils/locales';
+import i18n from '@/config/client/i18n';
+import * as usersApi from '@/modules/users/client/api/users.api';
+
+const SelectedLanguage = styled.strong`
+  padding: 7px 13px;
```

Comment:
Imitating button. Alternatively we give it `btn` class and overwrite some CSS. Might still do that since it might be easier to maintain.